### DISCUSSION TITLE
Load comment gifs in the  comment section

### DISCUSSION THREAD

**Discussion by Dre-OS** [Docile-Alligator/Infinity-For-Reddit#935] [Ideas]
I request to add this as an option, it would look better than the word "gif" being displayed.

**Comment by bczsalba** [comment]
Gif comments are a recent addition in the Reddit app, so I think this is something that will be added here sometime.

**Reply by anay-p** [reply]
Well, it's been a year so this feature isn't so recent anymore. Any progress on this? Tired of seeing 'gif' and 'sticker' in the comments.

**Comment by juliloving** [comment]
Boosting this post! It's annoying to see "gif"

**Reply by Wladefant** [reply]
To boost it upvote it

**Reply by flexagoon** [reply]
@Wladefant this was commended before this was converted into a discussion, and you can't upvote issues

**Reply by Wladefant** [reply]
did not saw it

**Comment by flexagoon** [comment]
+1, this would be a great feature to have

**Comment by KSPAtlas** [comment]
A sub I'm in recently enabled comment gufs and all the comments are gifs

**Reply by flexagoon** [reply]
Let me guess, r/WhenThe?

**Reply by KSPAtlas** [reply]
Yea

**Reply by flexagoon** [reply]
yep I came here for the same reason lol

**Comment by brij1234** [comment]
I would prefer if GIFs were “optional”. Loading GIFs requires some processing time, and it could result in some slowness. I know some other Reddit apps have a green button that you can click on to load the GIF.

**Reply by KSPAtlas** [reply]
Maybe an option in settings to pick between hide, button or always?

**Comment by patrickdemers6** [comment]
It would be nice to include images when doing this. Good use case would be in the sub `r/photoshopbattles`.

**Comment by anay-p** [comment]
@Docile-Alligator Any updates on this feature? Is it going to be implemented soon?

**Reply by ghost** [reply]
Would be pretty useful.

**Comment by loukwn** [comment]
Looks like it has been a while since we got a response from the devs so I decided to open a [PR](https://github.com/Docile-Alligator/Infinity-For-Reddit/pull/1474#issue-1718287835) as a suggestion. Feel free to put your comments.
